plugins/module_utils/foreman_helper.py

```diff
@@ -79,7 +88,7 @@ class KatelloMixin(OrganizationMixin):
     def connect(self):
         super(KatelloMixin, self).connect()
 
-        if 'subscriptions' not in self.foremanapi.resources:
+        if not self.has_plugin('katello'):
```

Comment:
Would it be possible to just add 'katello' to the `required_plugins`?

Reply:
Yeah, I considered that, would add ~5 lines and remove 2 XD

Reply:
implemented